ambry-server/src/integration-test/java/com.github.ambry.server/Verifier.java

```diff
@@ -163,6 +184,16 @@ public void run() {
                   if (userMetadataOutput.compareTo(ByteBuffer.wrap(payload.metadata)) != 0) {
                     throw new IllegalStateException();
                   }
+                  long expectedExpiryTimeMs = ServerTestUtil.getExpiryTimeMs(payload.blobProperties);
+                  long actualExpiryTimeMs =
+                      resp.getPartitionResponseInfoList().get(0).getMessageInfoList().get(0).getExpirationTimeInMs();
+                  if (actualExpiryTimeMs != expectedExpiryTimeMs) {
```

Comment:
could you add a helper method to do this expiry time check? I see this block of code in a few places.

Reply:
Done